fastlane/lanes/release.rb

```diff
@@ -44,10 +44,20 @@
     )
     ios_update_release_notes(new_version: new_version)
 
+    if prompt_for_confirmation(
+      message: 'Ready to push changes to remote to let the automation configure it on GitHub?',
+      bypass: ENV.fetch('RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM', nil)
+    )
+      push_to_git_remote(tags: false)
+      push_to_git_remote
```

Comment:
@mokagio Why is this used twice, one with the `tags` parameter and one without?

Reply:
Good catch @spencertransier! Thank you. 

This was a mistake. Not sure how I ended up with that, honestly. 🤔 Some copy-pasting / selecting gone wrong, maybe...

Reply:
```suggestion
      push_to_git_remote(tags: false)
```